projects/exui-common-lib/src/lib/models/filter.model.ts

```diff
@@ -21,6 +22,8 @@ export interface FilterConfig {
   cancelSetting?: FilterSetting;
   showCancelFilterButton?: boolean;
   cancelButtonCallback?(): void;
+  applyButtonCallback?(): void;
+  copyFields?(form: FormGroup): FormGroup;
```

Comment:
@aamirhm  does this need to be an optional function in the parameter?

Reply:
Yes, it is an optional function as only the user form needs a `copy` profile button.